src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/TelemetryHttpModule.cs

```diff
@@ -36,11 +48,37 @@ public class TelemetryHttpModule : IHttpModule
 
         private static readonly MethodInfo OnStepMethodInfo = typeof(HttpApplication).GetMethod("OnExecuteRequestStep");
 
+        private TraceContextPropagator traceContextPropagator = new TraceContextPropagator();
+
+        /// <summary>
+        /// Gets or sets the <see cref="TraceContextPropagator"/> to use to
+        /// extract <see cref="PropagationContext"/> from incoming requests.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TraceContextPropagator TextMapPropagator
```

Comment:
Do we need this? We can simply use `Propagators.DefaultTextMapPropagator` ?

Reply:
I think we do because `Propagators.DefaultTextMapPropagator` without an SDK will [no-op](https://github.com/open-telemetry/opentelemetry-dotnet/blob/6e765b8a2c9440de26f56aff0005049d445bb0e6/src/OpenTelemetry.Api/Context/Propagation/Propagators.cs#L32). In order to make this work as a standalone, I had to give it a default. Went with W3C because that is (more or less) what Microsoft.AspNet.TelemetryCorrelation does today. The existing one also looks for Request-Id & Correlation-Context so the new one isn't 100% compatible. I could add a custom default one to bridge the gap.

What I'm planning to do in our AspNet instrumentation is set this to `Propagators.DefaultTextMapPropagator` so whatever the SDK is using the module will respect.